src/Types/DelegationEvent.cs

```diff
@@ -27,6 +27,7 @@ internal static IDelegationEvent From(DelegationEvent delegationEvent) =>
                 DelegationAdded.From(delegationEvent.DelegationAdded),
             DelegationEvent.EventOneofCase.DelegationRemoved =>
                 DelegationRemoved.From(delegationEvent.DelegationRemoved),
+            DelegationEvent.EventOneofCase.BakerRemoved => throw new NotImplementedException(),
```

Comment:
Same here, why is this not implemented?

Reply:
It is implemented in https://github.com/Concordium/concordium-net-sdk/pull/102